lib/main.dart

```diff
@@ -7,119 +8,301 @@ void main() {
+                      ContainerGenres(text: Constants.genreAdventure),
+                      ContainerGenres(text: Constants.genreFantasy),
+                      ContainerGenres(text: Constants.genreComedy),
+                    ],
+                  ),
+                ),
+                const Padding(
+                  padding: EdgeInsets.all(Constants.paddingAllExtraInfo),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      Padding(
+                        padding:
+                            EdgeInsets.only(bottom: Constants.paddingOverview),
+                        child: Text(
+                          Constants.textOverview,
+                          style: TextStyle(
+                            fontWeight: FontWeight.bold,
+                            color: Colors.white,
+                          ),
+                        ),
+                      ),
+                      Padding(
+                        padding: EdgeInsets.only(
+                            bottom: Constants.paddingBottomOverview),
+                        child: Text(
+                          "While working underground to fix a water main, Brooklyn plumbers—and brothers—Mario and Luigi are transported down a mysterious pipe and wander into a magical new world. But when the brothers are separated, Mario embarks on an epic quest to find Luigi.",
+                          style: TextStyle(
+                            fontSize: Constants.fontSizeOverview,
+                            color: Colors.white,
+                          ),
+                        ),
+                      ),
+                      Padding(
+                        padding: EdgeInsets.only(
+                            bottom: Constants.paddingBottomOriginalTitle),
+                        child: Text(
+                          Constants.textOriginalTitle,
+                          style: TextStyle(
+                            color: Colors.white,
+                          ),
+                        ),
+                      ),
+                      Row(
+                        children: [
+                          Text(
+                            Constants.textLikeMovie,
+                            style: TextStyle(
+                              color: Colors.white,
+                            ),
+                          ),
+                          ButtonState()
+                        ],
+                      ),
+                    ],
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
       ),
-      home: const MyHomePage(title: 'Flutter Demo Home Page'),
     );
   }
 }
 
-class MyHomePage extends StatefulWidget {
-  const MyHomePage({super.key, required this.title});
+class ContainerGenres extends StatelessWidget {
```

Comment:
Lets move this widget to a different file

Reply:
Done!